lib/rules/landmark-has-body-context.js

```diff
@@ -1,7 +1,7 @@
 const nativeScopeFilter = 'article, aside, main, nav, section';
 
 // Filter elements that, within certain contexts, don't map their role.
-// e.g. a <footer> inside a <main> is not a banner, but in the <body> context it is
+// e.g. a <header> inside a <main> is not a banner, but in the <body> context it is
 return (
 	node.hasAttribute('role') ||
```

Comment:
Could this be extracted to `"selector": "[role]"`?

Reply:
I ended up not using this file, and the only thing I changed in it was a comment typo. Leaving as-is.

Reply:
I had to look it up too, but yes aside always maps to role= complementary, unlike footer and header which only map to a role if they are scoped to the body.